keps/sig-node/4191-split-image-filesystem/README.md

```diff
@@ -0,0 +1,955 @@
+
+###### What specific metrics should inform a rollback?
+
+If a cluster is evicting a lot more pods (`evictions`) than normal, this could be caused by this feature.
+
+The eviction manager monitors the image filesystem, node filesystem and the container filesystem for disk pressure.
+If any of these filesystems are experencing I/O pressure, pods will start being evicted and the eviction manager will trigger garbage collection.
+The metric `evictions` will inform operators that something is wrong because pods will be evicted and until disk pressure resolves itself, new workloads are not able to run.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+Not yet.
+
+We are testing with container runtimes not requiring this API implementation for initial alpha.
+
+In future releases, we could test this.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No.
+
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+
+This feature will be hidden from the users mostly but if an operator wants to know, it is possible to use [crictl](https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md).
+
+`crictl imagefsinfo` can be used to determine if the file systems are split.  
+
+`crictl imagefsinfo` will return a json object of file system usage for the image filesystem and the container filesystem.  If the image filesystem is not split, then the image filesystem and container filesystem will have identical statistics.
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [x] Other (treat as last resort)
+
+`crictl imagefsinfo` will give stats information about the different filesystems.
+
+A user could check the filesystem for containers and images.
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- [x] Metrics
+  - Metric name: Evictions
```

Comment:
is this name correct? It is the name you would use in Prometheus queries?

Reply:
I think so. I took the metric from here:

https://github.com/kubernetes/kubernetes/blob/f19b62fc0914b38941922afefd1e34eb55f87ee7/pkg/kubelet/metrics/metrics.go#L332C14-L332C14